tests/unit_tests/client/util/zip.cc

```diff
@@ -148,7 +149,8 @@ class ZIPFixtureImpl : ZIPFixture {
     std::vector<unsigned char> bad_bytes = good_bytes;
     std::fill_n(bad_bytes.begin() + offset, amount, byte);
     std::string str(bad_bytes.begin(), bad_bytes.end());
-    kovri::client::ZIP zip(str, str.size());
+    kovri::core::StringStream stream(str);
+    kovri::client::ZIP<kovri::core::StringStream> zip(&stream, str.size());
```

Comment:
To avoid reading the whole file, but it's quite stupid since this is required at some point in the impl.